### PR TITLE
webOS-ify

### PR DESCRIPTION
**Pull request by NoelMacwan**

### CODE REVIEW

File: README.mkdn [hunk 1]
```diff
@@ -1,50 +1,18 @@
-CyanogenMod
+webOS
 ===========
 
-Submitting Patches
```
Comment (morphis) [inline README.mkdn:4]: We should add a notice here that we're accepting pull-requests on github.com/webOS-ports for the repositories we've forked.

Comment (NoelMacwan) [inline README.mkdn:4]: Yeah,
I want to contact you via pm, can we?

**Comment by NoelMacwan** [comment]
so when you will accept this ?

File: README.mkdn (hunk 1 above)
Comment (morphis) [inline README.mkdn:4]: Just do if you want.

**Comment by morphis** [comment]
Add some description for the "Submitting Patches" section and I am fine.

**Comment by NoelMacwan** [comment]
Accept this pull request first then I can make another one for it ;)

**Comment by morphis** [comment]
No. Just push to the same branch as you did intially for this PR. It will extend the PR with the new commit. No need to create a new PR.Am 03.09.2014 16:14 schrieb Noel Macwan notifications@github.com: Accept this pull request first then I can make another one for it ;)

**Comment by NoelMacwan** [comment]
Okay then, I'll send new one soon

**Comment by NoelMacwan** [comment]
New PR sent !

**Pull request closed without merging**